internal/mode/static/state/graph/nginxproxy.go

```diff
@@ -186,7 +186,7 @@ func validateRewriteClientIP(npCfg *ngfAPI.NginxProxy) field.ErrorList {
 			default:
 				allErrs = append(
 					allErrs,
-					field.NotSupported(trustedAddressesPath.Child(addr.Value),
+					field.NotSupported(trustedAddressesPath.Child(string(addr.Type)),
```

Comment:
Sorry, I meant the literal string "type". This gives the user an idea of the field that is problematic.